python/ml4ir/base/features/feature_fns/tf_native.py

```diff
@@ -0,0 +1,59 @@
+import tensorflow as tf
+
+from ml4ir.base.io.file_io import FileIO
+
+
+def tf_native_op(feature_tensor: tf.Tensor, feature_info: dict, file_io: FileIO):
+    """
+    Run a series of tensorflow native operations on the input feature tensor.
+
+    Parameters
+    ----------
+    feature_tensor : Tensor
+        Input feature tensor
+    feature_info : dict
+        Dictionary representing the configuration parameters for the specific feature from the FeatureConfig
+    file_io : FileIO object
+        FileIO handler object for reading and writing
+
+    Returns
+    -------
+    Tensor object
+        Modified feature tensor after applying all the specified ops
+
+    Notes
+    -----
+    Args under feature_layer_info:
+        ops: list of dict
+            List of function specifications with associated arguments
+            
+            Arguments under opts:
+                fn : str
+                    Tensorflow native function name. Should start with tf.
+                    Example: tf.math.log or tf.clip_by_value
+                kwargs : dict
+                    Keyword arguments to be passed to the tensorflow function
+
+    * The functions will be applied in the order they are specified.
+    """
+    feature_node_name = feature_info.get("node_name", feature_info.get("name"))
+    tf_ops = feature_info.get("feature_layer_info", {}).get("args", {}).get("ops", {})
+
+    if not tf_ops:
+        return feature_tensor
+
+    for tf_op in tf_ops:
+        try:
+            fn_, kwargs = eval(tf_op["fn"]), tf_op.get("kwargs", {})
```

Comment:
I think the usage of `kwargs` is problematic here. This term has a somewhat special meaning in python, and what we are passing are not really `kwargs` but actual required arguments in these functions. You probably do it because we are passing them as a dictionary but I am afraid we are overloading and already overloaded term. Wouldn't `args` or `fn_args` make it simpler or at least minimize the risk of understanding something else?

Reply:
I see how you are using it below, I get it.

Reply:
but essentially these are not kwargs (at least not only kwargs)

Reply:
Yes, I can definitely rename to `args`